eq-author/src/App/questionPage/Design/Validation/withCustomValueChange.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from "prop-types";
```

Comment:
This could do with a better name as not all validation custom values are numeric and this is really specific to numeric fields.